jest.ci.json

```diff
@@ -2,9 +2,6 @@
     "testMatch": [
         "<rootDir>/src/**/__tests__/*.test.js"
     ],
-    "moduleFileExtensions": [
-        "js"
-    ],
```

Comment:
With this setting the tests on CI failes. Same happens locally when `yarn test-ci` is executed, while `yarn test` works (the only difference is this settings that prevents loading of the missing modules). The error in question:

> yarn run v1.22.4
> $ jest --config jest.ci.json
>  PASS  src/__tests__/stringUtils.test.js
>  PASS  src/__tests__/configBootstrap.test.js
>  PASS  src/__tests__/replaceInFiles.test.js
>  FAIL  src/__tests__/transpileTypeScriptFiles.test.js
>   ● transpileTypeScriptFiles › should transpile TypeScript files
> 
>     [BABEL]: Cannot find module 'core-js-compat/data' from 'node_modules/babel-plugin-polyfill-corejs3/lib/index.js'
> 
>     Require stack:
>       node_modules/babel-plugin-polyfill-corejs3/lib/index.js
>       node_modules/@babel/preset-env/lib/index.js
>       node_modules/@babel/core/lib/config/files/module-types.js
>       node_modules/@babel/core/lib/config/files/configuration.js
>       node_modules/@babel/core/lib/config/files/index.js
>       node_modules/@babel/core/lib/index.js
>       src/transpileTypeScriptFiles.js
>       src/__tests__/transpileTypeScriptFiles.test.js
>      (While processing: /Users/mort/dev/gooddata/gooddata-create-gooddata-react-app/node_modules/@babel/preset-env/lib/index.js)
> 
>       at Resolver.resolveModule (node_modules/jest-resolve/build/index.js:306:11)
>       at Object.<anonymous> (node_modules/babel-plugin-polyfill-corejs3/lib/index.js:6:36)
> 
> Test Suites: 1 failed, 3 passed, 4 total
> Tests:       1 failed, 17 passed, 18 total
> Snapshots:   0 total
> Time:        9.275 s, estimated 12 s
> Ran all test suites.
> error Command failed with exit code 1.
> info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.